cloud/scope/powervs_cluster.go

```diff
@@ -2492,32 +2490,6 @@ func (s *PowerVSClusterScope) deleteTransitGatewayConnections(tg *tgapiv1.Transi
 	return requeue, nil
 }
 
-// DeleteDHCPServer deletes DHCP server.
-func (s *PowerVSClusterScope) DeleteDHCPServer() error {
-	if !s.isResourceCreatedByController(infrav1beta2.ResourceTypeDHCPServer) {
```

Comment:
Do you think there will be scenario where user provides existing workspace and expects the DHCP server to be created? If so in that case we need to delete only DHCP server.

Reply:
True, we need to handle that. Will do the changes.

Reply:
Addressed this, ptal!